### PR TITLE
Remove hardcoded campaign

### PR DESCRIPTION
**Pull request by hakubo**
@bognix @Warkot please review

### CODE REVIEW

File: extensions/wikia/WikiaHomePage/WikiaHomePage.i18n.php [hunk 1]
```diff
@@ -16,7 +16,6 @@
 	'wikiahome-page-header-subheading' => '&nbsp;',
 	'wikiahome-page-header-create-wiki-button' => 'Start a Wiki',
 	'wikiahome-page-header-create-wiki-button-destination' => 'http://www.wikia.com/Special:CreateWiki',
-	'wikiahome-page-section-yearinfandom' => 'Celebrate and watch them all!',
```
Comment (hakubo) [inline extensions/wikia/WikiaHomePage/WikiaHomePage.i18n.php]: did grep for that, did not find anyone else using it

**Comment by Warkot** [comment]
I don't see this campaign on desktop anymore, so maybe just someone forgot to remove it form here. If guys from marketing are cool with dropping this, so am I.
LGTM :shipit:

**Comment by hakubo** [comment]
yup, they requested it
some months ago probably :)

**Comment by bkoval** [comment]
LGTM

**Pull request merged**